README.md

```diff
@@ -113,16 +124,27 @@ The tool should now be installed as `$GOPATH/bin/harbourbridge`
 To use the tool on a PostgreSQL database called mydb, run
 
 ```sh
-pg_dump mydb | $GOPATH/bin/harbourbridge
```

Comment:
Just wondering about whether we should remove the $GOPATH/bin prefix on the command here.
If your PATH environment variable is set up "properly" to include $GOPATH/bin, then this will work nicely.

But I'm not sure whether we can assume this?
I'm wondering if golang new users would have things set up this way (I found many golang quickstart guides saying to add /usr/local/go/bin to PATH, but none mentioned adding GOPATH/bin).
Maybe leaving $GOPATH/bin/harbourbridge is more foolproof: regular golang users who have their path set up properly will know they can drop this, so the extra verbosity isn't a huge issue.
Thoughts?

Reply:
Most of the users would have set up their PATH environment variable 'properly' and it is mentioned in installation guide as well ([Install GO](https://golang.org/doc/install)). I think it would be fair enough to assume this. Also, I have not come across go packages in which for running the tool they are using `$GOPATH/bin/tool_name` command. Mostly it is being left to the user to configure PATH properly.

Maybe we can make some update our README and explicitly specify to configure PATH. Below changes should be done in that case :

1. Update the following instruction in `Quickstart guide`/`Before you begin` section :
'configure the GOPATH environment variable if it is not already configured' --> 'configure the PATH environment variable to '/usr/local/go/bin' if it is not already configured'

2. Update the following instruction in `Quickstart guide`/`Installing HarbourBridge` section :
'The tool should now be installed as `$GOPATH/bin/harbourbridge`' --> 'The tool should now be installed as `harbourbridge` in your PATH environment variable'

what do you think?

Reply:
I think it is fine to keep this way. If I have configured `$GOPATH/bin/` in my `$PATH`, I would know that I can directly run `harbourbridge` command without the prefix `$GOPATH/bin/`. In the current way, it is more clear where the tool has been installed.

Reply:
Ok sure. I will undo current change.